omniconf/backends/argparse.py

```diff
@@ -92,26 +95,31 @@ def get_value(self, setting):
         * Settings with `_type=bool`, and where the default value is False will
           be specified as an argparse argument with `action=store_true`.
         """
```

Comment:
The part about the argparse interpretation, yes. But part above could be a little more clear, because we don't just parse a single Setting.